engines/wasmer/example_test.go

```diff
@@ -0,0 +1,30 @@
+package wasmer
+
+import (
+	"context"
+	"log"
+
+	"github.com/wasmerio/wasmer-go/wasmer"
+
+	"github.com/wapc/wapc-go"
+)
+
+// This shows how to customize the underlying wasmer engine used by waPC.
+func Example_custom() {
```

Comment:
note: this will end up in the godoc here once merged https://pkg.go.dev/github.com/wapc/wapc-go/engines/wasmer